TerminalDocs/search.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Windows Terminal Search
+description: Learn how to search in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: how-to
+ms.service: terminal
+---
+
+# How to search in Windows Terminal
+
+The Windows Terminal comes with a search feature that allows you to look through the text buffer for a specific keyword. This is useful when trying to find a command you had run before or for a specific file name.
+
+## Using search
+
+By default, you can open the search dialog by typing `ctrl+shift+f`. Once opened, you can type the keyword you're looking for into the text box and hit `enter` to search.
```

Comment:
Instead of using code blocks, does the docs page support the keybinding formatting? I think it's something like `<kbd> key </kbd>` to show <kbd> key </kbd>

Reply:
If it does, this might be worth doing a quick scan of the entire docs page to update these instances.